When running `python python-version/utils/script_which_complete_db.py`, if errors appear during downloads, it is because the links weren't updated according to this guideline: https://docs.google.com/document/d/1WnvA8gr82Y56Zya_UO43fFZF23vPOszR115IxnFleMU/edit